cmd/kubeadm/app/cmd/init.go

```diff
@@ -397,11 +392,11 @@ func (i *Init) Run(out io.Writer) error {
 
 	// PHASE 6: Install and deploy all addons, and configure things as necessary
 
-	if err := dnsaddonphase.EnsureDNSAddon(i.cfg, client); err != nil {
+	if err := proxyaddonphase.EnsureProxyAddon(i.cfg, client); err != nil {
```

Comment:
any particular reason you changed order here? I think it's okay but want to know the reasoning behind :)